staging/list_options.md

```
@@ -0,0 +1,46 @@
+# list_options - Configng V2 extra documents
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)